### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by AlekseyZaytcev**

### CODE REVIEW

File: Main.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
+require 'tty-prompt'
+require 'artii'
+require_relative 'logic/library'
+
+VERSION = '0.1.0'
```
Comment (valikos) [inline Main.rb]: Please rename to main.rb with lower case.

**Comment by valikos** [comment]
Please check all indentations. Replace tabs to whitespaces.

File: logic/library.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+  def top_books
+    result = ''
+    books = property_hash(:title, :book) { |var| var.sort_by do |key, value| value end }.reverse[0, 3]
+    books.each { |book| result += "\"#{get_book(book[0]).author.name}: #{book[0]}\" -> #{book[1]} orders\n" }
+    result
+  end
+
+  def save(file_name)
+    File.open(file_name, 'w') {|file| YAML.dump(self, file)}
```
Comment (valikos) [inline logic/library.rb]: 👍

File: logic/library.rb [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  end
+
+  def self.load(file_name)
+    YAML.load_file(file_name)
```
Comment (valikos) [inline logic/library.rb]: 👍

**Comment by AlekseyZaytcev** [comment]
I did it!

**Pull request merged**